Rephrase API settings (#264)
meilisearch/documentation: references/settings.md

#### Body

| Variable                 | Type      | Description                                                                      | Default value                                                                                     |
| ------------------------ | --------- | -------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- |

> | **synonyms**             | Object    | List of associated words that should be treated similarly in a search query      | `{}`                                                                                              |

Reviewer: ```suggestion
| **synonyms**             | Object    | List of associated words treated similarly      | `{}`                                                                                              |
```

> | **stopWords**            | [Strings] | List of words that should be excluded from search queries                        | `[]`                                                                                              |

Reviewer: ```suggestion
| **stopWords**            | [Strings] | List of words ignored by MeiliSearch when present in search queries                        | `[]`                                                                                              |
```

| **rankingRules**         | [Strings] | List of ranking rules sorted by order of importance                              | [A list of ordered built-in ranking rules](/guides/main_concepts/relevancy.md#order-of-the-rules) |

> | **distinctAttribute**    | String    | Returns only distinct (different) values of the given field                      | `null`                                                                                            |

Reviewer: ```suggestion
| **distinctAttribute**    | String    | Search returns documents with distinct (different) values of the given field                      | `null`                                                                                            |
```

All settings will be reset to their default value.

| Variable                 | Description                                                                      | Default value                                                                                     |
| ------------------------ | -------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- |

> | **synonyms**             | List of associated words that should be treated similarly in a search query      | `{}`                                                                                              |

Reviewer: ```suggestion
| **synonyms**             | List of associated words treated similarly  | `{}`                                                                                              |
```

> | **stopWords**            | List of words that should be excluded from search queries ignored                | `[]`                                                                                              |

Reviewer: ```suggestion
| **stopWords**            | List of words ignored by MeiliSearch when present in search queries                | `[]`                                                                                              |
```

| **rankingRules**         | List of ranking rules sorted by order of importance                              | [A list of ordered built-in ranking rules](/guides/main_concepts/relevancy.md#order-of-the-rules) |

> | **distinctAttribute**    | Returns only distinct (different) values of a given field                        | `null`                                                                                            |

Reviewer: ```suggestion
| **distinctAttribute**    | Search returns documents with distinct (different) values of a given field                        | `null`                                                                                            |
```
